packages/core/src/components/stepper/stepper.tsx

```diff
@@ -51,6 +51,7 @@ export class TdsStepper {
     }
   }
 
+  /** @internal Broadcasts changes to the Stepper props */
```

Comment:
I would change this to something like: Broadcasts changes in props to the stepper children.

Reply:
Nice, updated!